docs/usage/configuration.md

```diff
@@ -48,7 +48,7 @@ When the `gardener-controller-manager` starts it scans the `garden` namespace of
   * Not every end-user/stakeholder/customer has its own domain, however, Gardener needs to create a DNS record for every shoot cluster.
   * As landscape operator you might want to define a default domain owned and controlled by you that is used for all shoot clusters that don't specify their own domain.
 
-* **Alerting SMTP secrets** (optional), contain the SMTP credentials which will be used by the [AlertmMnager](https://prometheus.io/docs/alerting/alertmanager/) to send emails for alerts, please see [this](../../example/10-secret-alerting-smtp.yaml) for an example.
+* **Alerting SMTP secrets** (optional), contain the SMTP credentials which will be used by the [Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) to send emails for alerts, please see [this](../../example/10-secret-alerting-smtp.yaml) for an example.
```

Comment:
```suggestion
* **Alerting secrets** (optional), contain the SMTP credentials which will be used by the [Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) to send emails for alerts, please see [this](../../example/10-secret-alerting-smtp.yaml) for an example.
```

Can you also update the explanation? The alerting secrets can now contain more than just SMTP credentials, right?